index.js

```diff
@@ -1,39 +1,256 @@
+  if (commands.affirmative.includes(init.toLowerCase())) {
+    console.log(driveway.enterRoom());
+    player.currentScene = driveway
+    play()
+  }
+  else {
+    console.log("Fine! I didn't want to play either! ");
+    process.exit()
   }
+
 }
 
-start();
+async function play() {
+  let input = await ask('>_')
+  let lowerInput = input.toLowerCase()
+  let inputArray = lowerInput.split(' ');
+  let thisAction = inputArray[0];
+  let stash = inputArray[inputArray.length - 1]
+
+  if (lowerInput === 'exit') {
+    process.exit()
+  }
+
+  //scene inventory
+  else if (lowerInput === 'items') {
+    if (player.currentScene.inventory.length === 0) {
+      console.log("There is nothing here...")
+      play();
+    } else {
+      player.currentScene.inventory.forEach(item => console.log(item.name))
+      play();
+    }
+  }
+
+  //player inventory
+  else if (lowerInput === 'inventory') {
+    if (player.inventory.length === 0) {
+      console.log("You don't have anything...")
+      play();
+    } else {
+      player.inventory.forEach(item => console.log(item.name));
+      play();
+    }
+  }
 
-async function start() {
-  const welcomeMessage = `182 Main St.
-You are standing on Main Street between Church and South Winooski.
-There is a door here. A keypad sits on the handle.
-On the door is a handwritten sign.`;
-  let answer = await ask(welcomeMessage);
-  console.log('Now write your code to make this work!');
-  process.exit();
+  //movement and traps 
+  else if (commands.move.includes(thisAction)) {
+    if (inputArray.length === 1) {
+      console.log('maybe you should just sleep in the car');
+      play()
+    }
+    else {
+      let direction = stash;
+      if (player.currentScene[direction]) {
+        console.log(`Moving ${direction}...\n`);
+        player.changeScene(scenesTable[player.currentScene[direction]]);
+        console.log(player.currentScene.enterRoom())
+        play();
+      }
+      else if (direction !== 'forward' && direction !== 'backward' && direction !== 'right' && direction !== 'left') {
+        console.log("That's not a valid command\nPlease choose forward, backward, right, or left");
+        play()
+      }      
+      else if (currentScene = patio && direction === 'forward') {
+        console.log("You trip and fall\n Making a loud noise.\n Your wife wakes up.\n You die");
+        process.exit()
+      }
+      else if (currentScene = garden && direction !== 'left') {
+        console.log("\nOhhhhhhhh farts! You accidentally step on a stick causing a loud crack.\n A light turns on in the bedroom.\n You've awoken your wife.\n You die. ");
+        process.exit()
+      }
+      else if (currentScene = driveway && direction === 'left'|| 'backward') {
```

Comment:
Remember: you need to pass the whole truth check on each side of the logical operators. Currently 'backward' always evaluates to `true` since it's a string. It should be `direction === 'backward'` to make it a valid check.